blueprints/app/files/.eslintrc.js

```diff
@@ -43,11 +43,15 @@ module.exports = {
       env: {
         browser: false,
         node: true
-      }<% if (blueprint !== 'app') { %>,
+      },
       plugins: ['node'],
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
-        // add your custom rules and overrides for node files here
-      })<% } %>
+        // add your custom rules and overrides for node files here<% if (blueprint !== 'app') { %>
+
+        // this can be removed once the following is fixed
+        // https://github.com/mysticatea/eslint-plugin-node/issues/77
+        'node/no-unpublished-require': 'off'<% } %>
```

Comment:
This is because "everything" is a devDependencies?

Couldn't we fix by adding `files: ['dist']` or something similar to `package.json`?

Reply:
`files: ['dist']` might work, but would be a confusion addition to imply your app's dist should but published. Plus we can't put a comment to explain it in the json file.

Reply:
I was mistakenly thinking of no-extraneous-require's functionality, sorry for the confusion